webapp/packages/plugin-navigation-tree-rm/src/Tree/ProjectsRenderer/navigationTreeProjectsRendererRenderer.tsx

```diff
@@ -113,16 +112,14 @@ const ProjectRenderer: NavigationNodeRendererComponent = observer(function Manag
     );
   }
 
-  const project = node.nodeType === NAV_NODE_TYPE_RM_PROJECT && singleProject && !isDragging;
```

Comment:
This may break scripts/datasets trees (in these trees, we display fake projects instead of Scripts or Datasets folder)